apps/hashdotdev/package.json

```diff
@@ -26,13 +29,16 @@
     "@mui/system": "5.11.9",
     "axios": "0.27.2",
     "calculation-block": "0.2.2",
+    "chalk": "5.3.0",
```

Comment:
`devDependencies`? same for `execa`

Reply:
Addressed in https://github.com/hashintel/hash/pull/2768/commits/1065ad82546a1412903a3fd5435c59463ad73db2